### PR TITLE
Don't generate square and landscape poster images

### LINKED ISSUE #7849: Don't generate square and landscape poster images
## Bug Description

Currently, we automatically generate a landscape and square version of the poster from the portrait version that a user uploads. This can often result in incorrectly sized posters that are flagged as such in AMP Linter.

The ask is to only generate the landscape and square versions of the poster if they can adhere to the recommended size guidelines (1:1 for poster, 4:3 for landscape). If not, then do not generate the incorrectly sized posters for landscape and square and instead in the future provide options to the user to set these themselves.

## Expected Behaviour

Only generate the landscape and square versions of the poster if they can adhere to the recommended size guidelines.

## Steps to Reproduce

Add a poster to the Web Story and look at the auto-generated landscape and square versions of the same.

- Plugin Version: 1.7.2
- WordPress Version: 5.7.2
- Operating System: All
- Browser: All

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

* Remove square + landscape poster image sizes and their usage everywhere (incl.  AMP sanitization)

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Remove square and landscape poster images sizes and updated tests.

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7849

### CODE REVIEW

File: tests/phpunit/tests/Renderer/Story/HTML.php [hunk 1]
```diff
@@ -181,8 +181,6 @@ public function test_add_poster_images() {
 		$rendered = $this->setup_renderer( $post );
 
 		$this->assertContains( 'poster-portrait-src=', $rendered );
-		$this->assertContains( 'poster-square-src=', $rendered );
```
Comment (miina) [inline tests/phpunit/tests/Renderer/Story/HTML.php:184]: @spacedmonkey We also have some references to `poster-square` and `poster-landscape` in the edit-story folder.

Comment (spacedmonkey) [reply tests/phpunit/tests/Renderer/Story/HTML.php:184]: Good catch. Fixed now.

Comment (miina) [reply tests/phpunit/tests/Renderer/Story/HTML.php:184]: I think there's one more -- in the `mediaResponseWithPdfAndImage.json` file.

**Review by miina — APPROVED** [review]
Looks good pending https://github.com/google/web-stories-wp/pull/7947#discussion_r652522082

**Comment by spacedmonkey** [comment]
> Looks good pending #7947 (comment)

Fixed!

**Comment by swissspidy** [comment]
Did this go through QA to ensure the circles stories block still works as exoected?

**Comment by spacedmonkey** [comment]
> Did this go through QA to ensure the circles stories block still works as expected?

Worked fine in my local testing. 

This example is from 4 very landscape images. 
Screenshot 2021-06-18 at 10 22 39

One thing I did notice, is that source set or and lazy attribute were not added. I will create another ticket for those.

**Pull request merged**